.github/workflows/build.yml

```diff
@@ -183,7 +184,7 @@ jobs:
       - # === Sign Install Scripts (Windows only) ===
         name: Sign Install Scripts (Windows only)
         shell: powershell
-        if: runner.os == 'Windows' && contains(fromJSON('["refs/heads/master", "refs/heads/beta", "refs/heads/release"]'), github.ref)
+        if: runner.os == 'Windows' && contains(fromJSON('["refs/heads/master", "refs/heads/beta", "refs/heads/release", "refs/heads/LTS2022"]'), github.ref)
```

Comment:
This is going to get long as we have more and more LTS heads. How about instead of using `contains()`, using `github.event_name == 'push'`? Right now actions are run on `push` or `pull`, and all of the heads mentioned here are in the `push` event.